group-generators/helpers/data-providers/index.ts

```diff
@@ -186,10 +186,6 @@ export const dataProvidersAPIEndpoints = {
     getBeaconOwnersWithScoreCount: async (_: any) =>
       new DegenScoreProvider().getBeaconOwnersWithScoreCount(_),
   },
-  DiscourseProvider: {
```

Comment:
Why do you remove DiscourseProvider?

Reply:
> Why do you remove DiscourseProvider? But good catch it's not well implemented in the Factory, I'll make an update on the PR

I finally I tested it, and it doesn't work (because it's not fetching ethereum addresses).
I'll remove Discourse occurence in the PR